package.json

```diff
@@ -1,187 +1,197 @@
-    "webdriver:update": "node node_modules/protractor/bin/webdriver-manager update --standalone"
+    "webdriver:update": "node node_modules/protractor/bin/webdriver-manager update --standalone",
+    "lint": "tslint \"src/**/*.ts\" || true && tslint \"e2e/**/*.ts\" || true",
+    "docs": "typedoc --options typedoc.json ./src/",
+    "coverage": "http-server -c-1 -o -p 9875 ./coverage"
   },
   "dependencies": {
-    "@angular/common": "2.2.3",
-    "@angular/compiler": "2.2.3",
-    "@angular/compiler-cli": "2.2.3",
-    "@angular/core": "2.2.3",
-    "@angular/forms": "2.2.3",
-    "@angular/http": "2.2.3",
-    "@angular/platform-browser": "2.2.3",
-    "@angular/platform-browser-dynamic": "2.2.3",
-    "@angular/platform-server": "2.2.3",
-    "@angular/router": "3.2.3",
-    "@angular/upgrade": "2.2.3",
+    "@angular/animations": "4.2.6",
+    "@angular/common": "4.2.6",
+    "@angular/core": "4.2.6",
+    "@angular/forms": "4.2.6",
+    "@angular/http": "4.2.6",
+    "@angular/platform-browser": "4.2.6",
+    "@angular/platform-browser-dynamic": "4.2.6",
+    "@angular/platform-server": "4.2.6",
+    "@angular/router": "4.2.6",
     "@angularclass/bootloader": "1.0.1",
     "@angularclass/idle-preload": "1.0.4",
-    "@ng-bootstrap/ng-bootstrap": "1.0.0-alpha.18",
-    "@ngrx/core": "^1.2.0",
-    "@ngrx/effects": "2.0.2",
-    "@ngrx/router-store": "^1.2.5",
-    "@ngrx/store": "^2.2.1",
-    "@ngrx/store-devtools": "^3.2.2",
-    "@ngx-translate/core": "^6.0.1",
-    "@ngx-translate/http-loader": "^0.0.3",
-    "@types/jsonschema": "0.0.5",
-    "angular2-express-engine": "2.1.0-rc.1",
-    "angular2-platform-node": "2.1.0-rc.1",
-    "angular2-universal": "2.1.0-rc.1",
-    "angular2-universal-polyfills": "2.1.0-rc.1",
-    "body-parser": "1.15.2",
+    "@ng-bootstrap/ng-bootstrap": "1.0.0-alpha.28",
+    "@ngrx/core": "1.2.0",
+    "@ngrx/effects": "2.0.4",
+    "@ngrx/router-store": "1.2.6",
+    "@ngrx/store": "2.2.3",
+    "@nguniversal/express-engine": "1.0.0-beta.2",
+    "@ngx-translate/core": "7.0.0",
+    "@ngx-translate/http-loader": "0.1.0",
+    "body-parser": "1.17.2",
     "bootstrap": "4.0.0-alpha.6",
-    "cerialize": "^0.1.13",
-    "compression": "1.6.2",
-    "express": "4.14.0",
+    "cerialize": "0.1.15",
+    "compression": "1.7.0",
+    "cookie-parser": "1.4.3",
+    "core-js": "2.4.1",
+    "express": "4.15.3",
+    "express-session": "1.15.3",
     "font-awesome": "4.7.0",
-    "http-server": "^0.9.0",
+    "http-server": "0.10.0",
+    "https": "1.0.0",
     "js.clone": "0.0.3",
-    "jsonschema": "^1.1.1",
+    "jsonschema": "1.1.1",
     "methods": "1.1.2",
-    "morgan": "1.7.0",
-    "ng2-pagination": "^2.0.0",
-    "preboot": "4.5.2",
-    "reflect-metadata": "^0.1.10",
-    "rxjs": "5.0.0-beta.12",
-    "ts-md5": "^1.2.0",
-    "webfontloader": "1.6.27",
-    "zone.js": "0.6.26"
+    "morgan": "1.8.2",
+    "ng2-pagination": "2.0.2",
```

Comment:
Now we can update with 'ngx-pagination' package.
It should work properly, the only thing to do is to remove [these lines](https://github.com/wwelling/dspace-angular/blob/c08f5c672b1f848d1cf62f275269cd8e2b76616e/src/app/shared/pagination/pagination.component.ts#L21,L22)